api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -708,24 +710,192 @@ default void openBook(final @NotNull Book book) {
   // ------------------------
 
   /**
-   * Sends a resource pack request to this audience.
+   * Sends resource pack requests to this audience, replacing any existing resource packs that might be present.
+   *
+   * <p>Multiple resource packs are only supported since 1.20.3. On older versions, only the first pack will be sent.</p>
+   *
+   * @param request the first resource pack request
+   * @param others other requests
+   * @see ResourcePackRequest
+   * @since 4.15.0
+   */
+  @SuppressWarnings("checkstyle:MethodName")
+  @ForwardingAudienceOverrideNotRequired
+  default void setResourcePack(final @NotNull ResourcePackRequestLike request, final @NotNull ResourcePackRequestLike@NotNull... others) {
```

Comment:
It does not make sense to accept more than one pack request for a "setResourcePack" method imo. If you want to have multiple, you use the send methods. set should just be a single one. Maybe making the name setResourcePacks would be better.

Reply:
The difference between set and send is, set is (clear + send), so why wouldn't it accept multiple. Making the method name plural would make sense I suppose.